packages/zoe/test/swingsetTests/zoe/vat-alice.js

```diff
@@ -52,10 +52,15 @@ const build = async (log, zoe, issuers, payments, installations, timer) => {
       UnderlyingAsset: moolaIssuer,
       StrikePrice: simoleanIssuer,
     });
-    const { creatorInvitation: writeCallInvitation } = await E(
+    const { creatorInvitation: writeCallInvitation, creatorFacet } = await E(
       zoe,
     ).startInstance(installation, issuerKeywordRecord);
 
+    const adminNode = E(creatorFacet).getAdminNode();
+    E(adminNode)
+      .done()
+      .catch(e => log(`covered call was shut down due to ${e}`));
+
```

Comment:
Sorry for the multiple comments, but I think making `adminNode` available just for testing is the wrong thing to do here. We could use the golden tests to test.